Makefile

```diff
@@ -268,9 +268,9 @@ localnet-start: localnet-stop
 	docker rmi -f ethermint-build-linux
 	docker build --rm -t ethermint-build-linux . ; \
 	container_id=$$(docker create ethermint-build-linux) ; \
-	docker cp $${container_id}:/usr/bin/emintd ./build/ ; \
-	docker cp $${container_id}:/usr/bin/emintcli ./build/
```

Comment:
so this copyies the binaries to the build folder, which allows the `./build:/emintd:Z` volumes to be mounted.

Reply:
is it necessary for the binary to be copied over to the directory? Why don't we have the binary just be built into the image directly?

This is how i do that:
https://github.com/ChainSafe/ethermint/blob/fix-testnet/Dockerfile#L25-L26